dmoj/result.py

```diff
@@ -66,6 +68,24 @@ def set_result_flag(self, process):
         if process.mle:
             self.result_flag |= Result.MLE
 
+    def update_feedback(self, error, process, binary, feedback=None):
```

Comment:
Perhaps to make this less funky to use, it should be a something like `@classmethod get_feedback_str`, since the `get_main_code` check doesn't actually need the entirety of `set_result_flag`.

I propose adding `@property ir` and `@property rte` into the process class. (It would be nice if `tle` and `mle` could be renamed to `is_tle` and `is_mle`, but that can be done in the future.)

Doing all this would clean up the hack of constructing a dummy `Result` object, and give this routine a better name.